tools/Jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
```

Comment:
Is a license header required?  I think yes.

Reply:
Right, I added it.

Reply:
@maht do we still need this patch, if we you github action to do precheck:
https://github.com/apache/incubator-nuttx/pull/261

Reply:
@xiaoxiang781216 No, we don't need it anymore. It can be closed.